### ISSUE TITLE
Concerns with ethnicity data collection

### ISSUE THREAD

**Issue by rdpiazza** [uswds/uswds#5762]
## Description

I appreciate the USWDS team's efforts to provide race and ethnicity patterns that better capture a modern understanding of these topics. At the same time, I am concerned that the recommended patterns may lead to changes in data collection that do not accurately represent the public being served. 

Specifically, on the page providing [Race and ethnicity guidance](https://designsystem.digital.gov/patterns/create-a-user-profile/race-and-ethnicity/), the recommended pattern for capturing ethnicity is an open-ended text field. Per OMB guidance cited on the page, ethnicity information is used in Federal statistical, administrative, and compliance reporting to capture information about how much of the population served is Hispanic or Latino or not Hispanic or Latino. 

Has this open-ended text pattern undergone testing with Hispanic and Latino users to validate that self-reporting of Hispanic and Latino data is equivalent using this pattern vs. a pattern focusing on the two OMB-approved ethnicity options? I am deeply concerned that the open-ended text box for ethnicity will lead to under-reporting by Hispanic and Latino respondents, which could have significant outcomes in program administration, outreach, and the ability of programs to understand and better meet the needs of the populations they serve. 

The US Digital Service worked with USDA Food and Nutrition Service to research how to best ask "sensitive questions" (including race and ethnicity) of participants in the WIC program. In their research, they found that ethnicity was the question that caused the most confusion, and that was with the additional clarity provided by radio-button options. That research makes me concerned about the ability to accurately capture the Hispanic/Latino information needed for statistical analysis from a free-text field. 

What research was performed to validate the statistical accuracy of self-reported ethnicity data via the open-ended field pattern vs. one with predefined options of Hispanic or Latino/Not Hispanic or Latino? If this was not tested and validated for statistical accuracy, I strongly recommend changing the pattern so that data collections across government are not put at risk for underreporting Hispanic and Latino participation.

**Comment by annepetersen** [comment]
Hi Rebecca: thanks for bringing this up. Publishing this was based on [research we conducted](https://designsystem.digital.gov/together/#section_3), but also we understand that use cases across government vary quite a bit and that things change over time. 

In fact, we [recently spoke at an event](https://www.youtube.com/watch?v=q08f2IE0OrA) where we recommended maybe adding another field to clarify further (section starts at 22:11, most relevant portion is perhaps at 29:00 or so), showing a vote.gov example. 

We'll take a closer look at whether or not we should update the guidance. If we have questions regarding your take here, we'll reach out. Thanks again!
